### PR TITLE
CotE - Glorious Victory

### PR DESCRIPTION
**Pull request by SArendeli**
My first card contribution...think this should work, but definitely warrants checking!

### CODE REVIEW

**Comment by SArendeli** [comment]
Merged implementation and test branches

File: server/game/cards/06-CotE/GloriousVictory.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+class GloriousVictory extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            title: 'Honor each character you control',
+            when: {
+                onBreakProvince: (event, context) => this.game.isDuringConflict('military') && event.conflict.attackingPlayer === context.player
+            },
+            gameAction: ability.actions.honor(context => ({
+                target: this.game.findAnyCardsInPlay(card => card.getType() === CardTypes.Character && card.controller === context.player)
```
Comment (jamesoddy) [inline server/game/cards/06-CotE/GloriousVictory.js]: Not a big deal, but this is a slightly cleaner version of this line
```
target: context.player.filterCardsInPlay(card => card.getType() === CardTypes.Character)
```

File: test/server/cards/06-CotE/GloriousVictory.spec.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            it('should not trigger on breaking a province during a political conflict', function() {
+                this.initiateConflict({
+                    type: 'political',
+                    province: this.shamefulDisplay,
+                    attackers: [this.shinjoAltansarnai],
+                    defenders: [],
+                    jumpTo: 'afterConflict'
+                });
+                this.player1.clickPrompt('No');
+                expect(this.player1).not.toHavePrompt('Triggered Abilities');
```
Comment (jamesoddy) [inline test/server/cards/06-CotE/GloriousVictory.spec.js]: Again, not a big deal, but I would add this line at the end of this test:
```
expect(this.player1).toHavePrompt('Air Ring');
```
This is just to check that you have passed the 'break province' step and Glorious Victory has not triggered.

File: test/server/cards/06-CotE/GloriousVictory.spec.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+            });
+
+            it('should trigger when Endless Plains fires', function() {
+                this.initiateConflict({
+                    type: 'military',
+                    province: this.endlessPlains,
+                    attackers: [this.shinjoAltansarnai],
+                    defenders: []
```
Comment (jamesoddy) [inline test/server/cards/06-CotE/GloriousVictory.spec.js]: `defenders: []` needs to be removed as `this.initiateConflict` will error as triggers are occuring before the declare defender step.

File: test/server/cards/06-CotE/GloriousVictory.spec.js (hunk 2 above)
Comment (SArendeli) [reply test/server/cards/06-CotE/GloriousVictory.spec.js]: Added

**Pull request merged**